skl2onnx/operator_converters/logistic_regression.py

```diff
@@ -0,0 +1,88 @@
+# -------------------------------------------------------------------------
```

Comment:
I'm wondering why you need to create a separate converter instead of fixing the current one which is the direction I followed in the other PR #150.

Reply:
I think we can go ahead with your PR. For some reason I saw some issues when I tried to fix the linear_classifier converter.

Reply:
I let you review it then.